### ISSUE TITLE
Clarify objective of this repository in the README

### ISSUE THREAD

**Issue by j4yk** [oasp/oasp4j-ide#77]
Hi, I would like to create a new application, possibly using OASP, and I am in the process of evaluating if I need the IDE part or not. I have not yet figured out, whether this repository is mainly about the automatic, reproducable generation of development environments (such as Eclipse workspaces) or if it provides a set of packages/software/plugins/... that are specifically helpful for OASP development. Digging through the wiki I found that I need a scripts- and a software package. But as far as I can see, no such software package is provided for download, so would I have to provision my Eclipse with the required plugins anyway, at least once? When I participated in an oasp/devon TP in October last year, we got an all-in-one download, but I figure this is not suitable to direct any future maintainers to.

In short, I would like the README to clarify what this repository provides to a newcomer and what must be done to get started.

By the way, wouldn't it be beneficial to include a link to the wiki pages there as well, or move the must-read content to the README directly? It is the first (or second) thing a GitHub user sees when they are browsing through the repositories.

**Comment by hohwille** [comment]
Of course documentation can always be improved. As a very first step, I added a link to the wiki in the README. Next we can still improve the wiki docs.
We can not provide the software package as free OSS download on the web due to legal issues.
Therefore we provide instructions to build a software package manually:
https://github.com/oasp/oasp4j-ide/wiki/guide-software-package

Further we are trying to provide a solution based on OOMPH that can automatically download and build the software packge for you.

Finally I you are working for Capgemini simply use the link provided in our instructions:
https://github.com/oasp/oasp4j/wiki/oasp-ide-setup

**Comment by j4yk** [comment]
Thank you, I was not aware of the last article. Would you mind if I copied the instructions in item *2. Download the IDE* over to the install instructions in this repository's wiki?

**Comment by hohwille** [comment]
> Would you mind if I copied the instructions in item 2. Download the IDE over to the install instructions in this repository's wiki?

This does not belong here. This repository is providing a "generic" tool that can be used with OASP or without it can be used at Capgemini or by anybody else in the world. To me the question is more, why did you come here and what did you expect?
IMHO we should close this issue...

**Comment by j4yk** [comment]
Originally, I expected this repository to provide a (ready-to-use) IDE tailored for OASP4J development (like an Eclipse distrubution plus additional tools, maybe). But as you pointed out, that is not the case (strictly).

In my opinion, the repository name oasp4j-ide suggests my expectation, while you could deduce from the Readme that it is something different. And the Wiki says that this IDE tooling is part of OASP4J, hence the repository name... But I had to take some turns to get that. That's why I requested to clarify the Readme, so newcomers will be able grasp more quickly what they will find here and what not.

I am not sure if this is something only I got wrong or if somebody else will get this wrong in the future. If you doubt that, feel free to close the issue.

**Comment by hohwille** [comment]
Updated Wiki start-page that is linked from README with an ATTENTION and a link to IDE setup.
